tests/integration/contributors_test.py

```diff
@@ -409,3 +411,68 @@ def test_delete_contributor(test_client, with_custom_contributors):
     # Check that a deleted contributor can't be deleted again
     delete_resp = test_client.delete("/contributors/{}".format(contributor_to_delete))
     assert delete_resp._status_code == 404
+
+
+def test_purge_contributor(test_client, basic_gtfs_rt_data, mock_rabbitmq):
+    """
+    Test the purge with different contributors and rt data
+    """
+
+    def test_rt_data():
+        with app.app_context():
+            assert len(RealTimeUpdate.query.all()) == 1
+            assert len(TripUpdate.query.all()) == 1
+            assert len(VehicleJourney.query.all()) == 1
+            assert len(StopTimeUpdate.query.all()) == 4
+
+    def test_contributor_count(contrib_count):
+        with app.app_context():
+            assert len(Contributor.query.all()) == contrib_count
+
+    def deactivate_contributor(contributor_id):
+        with app.app_context():
+            contrib = Contributor.query.filter_by(id=contributor_id).first()
+            contrib.is_active = False
+            db.session.commit()
+
+    def has_rt_data(contributor_id):
+        with app.app_context():
+            return len(RealTimeUpdate.query.filter_by(contributor_id=contributor_id).all()) > 0
+
+    # We have 4 contributors and all are active:
+    test_contributor_count(4)
+    with app.app_context():
+        contributors = Contributor.query.all()
+        assert contributors[0].id == "rt.tchoutchou"
+        assert contributors[1].id == "rt.vroumvroum"
```

Comment:
Minor : risky if an other contributor is added for an other test.
```suggestion
        contributors_id = [c.id for c in contributors]
        assert "rt.vroumvroum" in contributors_id
        assert "rt.tchoutchou" in contributors_id
```